code/ObjFileParser.cpp

```diff
@@ -642,7 +638,7 @@ void ObjFileParser::getMaterialLib() {
     m_pIO->Close( pFile );
 
     // Importing the material library
-    ObjFileMtlImporter mtlImporter( buffer, strMatName, m_pModel );
+    ObjFileMtlImporter mtlImporter( buffer, strMatName, std::move(m_pModel) );
```

Comment:
I'm pretty sure this is wrong and the cause of the crash in the test suite. You don't want to pass the ownership of `m_pModel`, just borrow the pointer with ´.get()´. That means that the ObjFileMtlImporter constructor should remain unchanged.

Reply:
Yeah you're right, I have submitted the revision too early :/